### DISCUSSION TITLE
Unable to fork a repository

### DISCUSSION THREAD

**Discussion by federal1970** [community/community#112404] [New to GitHub]
Hi there!

I've faced with an issue forking repository.
I'm trying to fork this repo https://github.com/jonasschmedtmann/ultimate-react-course

but after clicking [Create fork] button a page gets redirected to the original repo without making a fork in my account.

Could you please help?

**Comment by keeper-22** [comment]
I have this same issue

**Reply by dmore** [reply]
not working for me either

**Comment by developer-sumit** [comment]
If you're experiencing issues forking a repository on GitHub, there could be a few reasons why this is happening. Here are some steps you can take to troubleshoot the issue:

1. **Check Permissions**: Ensure that you have the necessary permissions to fork repositories on GitHub. Make sure you are logged in to your GitHub account and that you have the appropriate access level (e.g., you are not part of an organization that restricts forking).

2. **Try a Different Browser or Device**: Sometimes, browser extensions or settings can interfere with GitHub's functionality. Try using a different browser or device to see if the issue persists.

3. **Clear Browser Cache and Cookies**: Cached data or cookies in your browser could be causing the problem. Try clearing your browser's cache and cookies and then attempt to fork the repository again.

4. **Check GitHub Status**: Occasionally, GitHub may experience service interruptions or issues. Check the GitHub Status page (https://www.githubstatus.com/) to see if there are any ongoing incidents that might be affecting repository forking functionality.

5. **Network Connection**: Ensure that your internet connection is stable. A poor or intermittent connection could prevent the fork operation from completing successfully.

6. **GitHub Support**: If none of the above steps resolve the issue, you may want to reach out to GitHub Support for further assistance. They can help investigate the problem and provide guidance on how to resolve it.

If you've tried these steps and are still unable to fork the repository, it's possible that there may be an issue with the repository itself or with your account settings. In such cases, contacting GitHub Support would be the best course of action to resolve the issue. Remember to mark it as "answered" if you like my answer

**Reply by eyalroz** [reply]
I am also facing this issue (with https://github.com/thunderbird/webext-support). I tried all of (1.) through (5.) and nothing worked.

**Reply by armchairancap** [reply]
> I am also facing this issue (with https://github.com/thunderbird/webext-support). I tried all of (1.) through (5.) and nothing worked.

I can fork that repo as well as the one at the very top. Maybe the repos was changed since the problem happened, or maybe it's just your account.

I used to own a complex repo (external submodules, etc.) forked from another repo GH repo. 
As I added and removed commits, at one point it became impossible to fork and even Releases tab was no longer visible.

After weeks of trying to figure it out I gave up and removed most of its content that I didn't need as well as all git history, effectively making a new repo in existing location... I wouldn't call that a "fix" but that's how it went. I can't tell what it was, but I think it was a some kind of GH back-end bug.

**Comment by dylan-chong** [comment]
so, what fixed it for you @federal1970 @keeper-22

**Comment by keeper-22** [comment]
When I signed up GitHub put some restrictions on my account for some reason. I had to contact them to lift the restriction. Very frustrating.
